docs/root/configuration/http/http_filters/router_filter.rst

```diff
@@ -371,7 +371,7 @@ owning HTTP connection manager.
   :widths: 1, 1, 2
 
   no_route, Counter, Total requests that had no route and resulted in a 404
-  no_cluster, Counter, Total requests in which the target cluster did not exist and resulted in a 404
+  no_cluster, Counter, Total requests in which the target cluster did not exist and which by default result in a 503
```

Comment:
I'd just say `and resulted in a 503`. "by default" makes it sound like a configurable option.

Reply:
It is configurable:

https://github.com/envoyproxy/envoy/blob/master/api/envoy/api/v2/route/route_components.proto#L719

Reply:
doh TIL